addon/index.js

```diff
@@ -265,7 +245,7 @@ function handleToolbarButtonChange(state) {
   panel.show({
     width: PANEL_WIDTH,
     height: (experimentCount * EXPERIMENT_HEIGHT) + FOOTER_HEIGHT,
-    position: button
+    position: {top: 2, right: 2}
```

Comment:
Hmm, this might just be my personal edge case nit, but the panel doesn't appear near the button with this change. My toolbar's a bit rearranged via customize, so this ends up on the other side of the browser from where I clicked the button.